docs/readme/readme_bn.md

```diff
@@ -41,12 +41,11 @@ choco install bruno
 snap install bruno
 
 # Apt এর মাধ্যমে লিনাক্সে
-sudo mkdir -p /etc/apt/keyrings
+sudo mkdir -p /etc/apt/keyrings /root/.gnupg
 sudo gpg --no-default-keyring --keyring /etc/apt/keyrings/bruno.gpg --keyserver keyserver.ubuntu.com --recv-keys 9FA6017ECABE0266
-
 echo "deb [signed-by=/etc/apt/keyrings/bruno.gpg] http://debian.usebruno.com/ bruno stable" | sudo tee /etc/apt/sources.list.d/bruno.list
-
 sudo apt update
+sudo apt install libasound2
```

Comment:
Instead of adding the package to the docs, we can just add it as a dependency to the package in the [build config](https://github.com/usebruno/bruno/blob/main/packages/bruno-electron/electron-builder-config.js): https://www.electron.build/configuration/linux#debian-package-options

Reply:
I don't have enough knowledge myself to implement this, but idea is right - hope someone can add this 🙏

Reply:
I just tested it locally, adding this into https://github.com/usebruno/bruno/blob/main/packages/bruno-electron/electron-builder-config.js after line 36 works. Can you add this to you're PR?
```js
  deb: {
    // Docs: https://www.electron.build/configuration/linux#debian-package-options
    depends: [
      "libgtk-3-0",
      "libnotify4",
      "libnss3",
      "libxss1",
      "libxtst6",
      "xdg-utils",
      "libatspi2.0-0",
      "libuuid1",
      "libsecret-1-0",
      "libasound2" // #1036
    ],
  },
```

```
❯ dpkg -I packages/bruno-electron/out/bruno_1.2.0_amd64_linux.deb
 new Debian package, version 2.0.
 size 69392000 bytes: control archive=3055 bytes.
     474 bytes,    14 lines      control
    5236 bytes,    84 lines      md5sums
     265 bytes,    10 lines   *  postinst             #!/bin/bash
      68 bytes,     4 lines   *  postrm               #!/bin/bash
 Package: bruno
 Version: 1.2.0
 License: unknown
 Vendor: Anoop M D <anoop.md1421@gmail.com>
 Architecture: amd64
 Maintainer: Anoop M D <anoop.md1421@gmail.com>
 Installed-Size: 272070
 Depends: libgtk-3-0, libnotify4, libnss3, libxss1, libxtst6, xdg-utils, libatspi2.0-0, libuuid1, libsecret-1-0, libasound2
 Recommends: libappindicator3-1
 Section: default
 Priority: optional
 Homepage: https://www.usebruno.com
 Description:
   Opensource API Client for Exploring and Testing APIs
```

Reply:
Sure, I pushed new commits!

But I wasn't able to reproduce package generation - when I run:

```bash
npm run dist:deb
```

I got:

```
...
> bruno@v1.2.0 dist:deb
> electron-builder --linux deb --config electron-builder-config.js

  • electron-builder  version=23.0.2 os=10.0.22631
  • loaded configuration  file=E:\Forks\bruno\packages\bruno-electron\electron-builder-config.js
  • writing effective config  file=out\builder-effective-config.yaml
  • packaging       platform=linux arch=x64 electron=21.1.1 appOutDir=out\linux-unpacked
  • scheduling remote build  target=deb arch=x64
  • cannot get, wait  error=Get "https://service.electron.build/find-build-agent?no-cache=1ilvbkt": dial tcp 51.15.76.176:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
                      attempt=0
                      waitTime=2
  • cannot get, wait  error=Get "https://service.electron.build/find-build-agent?no-cache=1ilvbkt": dial tcp 51.15.76.176:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
                      attempt=1
                      waitTime=4
  • cannot get, wait  error=Get "https://service.electron.build/find-build-agent?no-cache=1ilvbkt": dial tcp 51.15.76.176:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
                      attempt=2
                      waitTime=6
  ⨯ Get "https://service.electron.build/find-build-agent?no-cache=1ilvbkt": dial tcp 51.15.76.176:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
  ⨯ E:\Forks\bruno\node_modules\app-builder-bin\win\x64\app-builder.exe process failed ERR_ELECTRON_BUILDER_CANNOT_EXECUTE
Exit code:
1  failedTask=build stackTrace=Error: E:\Forks\bruno\node_modules\app-builder-bin\win\x64\app-builder.exe process failed ERR_ELECTRON_BUILDER_CANNOT_EXECUTE
...
```

I am not familiar with Electron, but I will try to investigate it later...

Reply:
Okay, I successfully overcome this by using docker container as mentioned [here](https://github.com/electron-userland/electron-build-service/issues/9).